client/src/components/AddressToolbar.tsx

```diff
@@ -1,11 +1,25 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import Modal from 'components/Modal';
+import Modal from './Modal';
 
 import 'styles/AddressToolbar.css';
 
-export default class AddressToolbar extends Component {
-  constructor(props) {
+export type AddressToolbarProps = {
+  onExportClick: () => void;
+  numOfAssocAddrs: number;
+  userAddr: {
+    housenumber: string;
+    streetname: string;
+    boro: string;
```

Comment:
I suspect we will eventually separate this data type into a different place, because I've seen it elsewhere, but that can wait for a future PR where we actually reuse the pattern in a different component.